src/l500/ac-trigger.cpp

```diff
@@ -681,12 +696,27 @@ namespace ivcam2 {
 
                     auto df = _sf.get_depth_frame();
                     auto irf = _sf.get_infrared_frame();
-                    depth_to_rgb_calibration algo( df, irf, _cf, _pcf, cal_info, cal_regs );
+
+                    depth_to_rgb_calibration algo(df, irf, _cf, _pcf, cal_info, cal_regs, 
+                    [&]() 
+                    {
+                        if (!is_processing())
+                        {
+                            AC_LOG( DEBUG , "Stopping CAH algorithm");
```

Comment:
"stopping algo: not processing any more"

Reply:
Done